README.md

```diff
@@ -46,9 +46,9 @@ Notice:  Constant BBBB does not exist in self scope in fixtures/simple/undefined
 
     return self::BBBB; 
 
-Notice:  You are trying to cast 'string' to 'string'. in fixtures/simple/code-smell/StandardFunctionCall.php on 16 [stupid.cast]
+Notice:  You are trying to cast 'string' to 'string' in fixtures/simple/code-smell/StandardFunctionCall.php on 16 [stupid.cast]
 
-    return (string) json_encode(array(
+    return (string) json_encode(array)
```

Comment:
This is the line triggering the notice so `return (string) json_encode(array(` is actually correct.